src/test/java/com/exactpro/epfast/template/TemplateAssertTests.java

```diff
@@ -0,0 +1,42 @@
+package com.exactpro.epfast.template;
+
+import com.exactpro.epfast.template.assertion.TemplateAssert;
+import com.exactpro.epfast.template.simple.Identity;
+import com.exactpro.epfast.template.simple.Reference;
+import com.exactpro.epfast.template.simple.Template;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class TemplateAssertTests {
+
+    private Template temp = new Template();
```

Comment:
If you perform other initialization steps in constructor, it's better to move construction of these member there as well. It will make code easier to read as all related code will be in one place.